setup.py

```diff
@@ -35,14 +35,11 @@
         'dev': [
             "coverage==4.2",
             "flake8==3.2.1",
-            "mccabe==0.6.1",
-            "pycodestyle==2.3.1",
-            "pyflakes==1.5.0",
             "pylint==1.7.1",
             "pytest==2.7.3",
             "pytest-cov==2.1.0",
             "pytest-timeout==0.4",
-            "urllib3==1.10.2",
+            "urllib3==1.21.1",
```

Comment:
Until very recently (2.16.0) requests did vendor urllib3, although [here](https://github.com/nameko/nameko/blob/0c44614bd1eb93c00977bb48169d0245a25eb453/test/test_amqp.py#L7) we are using it directly anyway.

I will pin requests here in `[dev]` so everything is consistent, and bump all the versions to something recent.